mtimmara-bengaluru.md

```diff
@@ -0,0 +1,7 @@
+
+
+### Places to Eat
+ - Vidhyarthi Bhavan
+ - Shivagi Military Hotel
+ - Taza Thindi

```

Comment:
I have no clue what any of these places are but I am always up for an adventure!